rdr_service/resource/generators/genomics.py

```diff
@@ -0,0 +1,177 @@
+#
+# This file is subject to the terms and conditions defined in the
+# file 'LICENSE', which is part of this source code package.
+#
+#
+# This file is subject to the terms and conditions defined in the
+# file 'LICENSE', which is part of this source code package.
+#
+from sqlalchemy.sql import text
+
+from rdr_service.dao.resource_dao import ResourceDataDao
+from rdr_service.participant_enums import (
+    GenomicSetStatus as GenomicSetStatusEnum,
+    GenomicSetMemberStatus as GenomicSetMemberStatusEnum,
+    GenomicValidationFlag as GenomicValidationFlagEnum,
+    GenomicSubProcessStatus as GenomicSubProcessStatusEnum,
+    GenomicSubProcessResult as GenomicSubProcessResultEnum,
+    GenomicJob as GenomicJobEnum,
+    GenomicWorkflowState as GenomicWorkflowStateEnum
+)
+from rdr_service.resource import generators, schemas
+
+
+class GenomicSetSchemaGenerator(generators.BaseGenerator):
+    """
+    Generate a GenomicSet resource object
+    """
+
+    def make_resource(self, _pk, backup=False):
+        """
+        Build a resource object from the given primary key id.
+        :param _pk: Primary key value from rdr table.
+        :param backup: if True, get from backup database instead of Primary.
+        :return: resource object
+        """
+        ro_dao = ResourceDataDao(backup=backup)
+        with ro_dao.session() as ro_session:
+            row = ro_session.execute(text('select * from genomic_set where id = :id'), {'id': _pk}).first()
+            data = ro_dao.to_dict(row)
+
+            # Populate Enum fields.
+            if data['genomic_set_status']:
+                enum = GenomicSetStatusEnum(data['genomic_set_status'])
+                data['genomic_set_status'] = str(enum)
+                data['genomic_set_status_id'] = int(enum)
+
+            return generators.ResourceRecordSet(schemas.GenomicSetSchema, data)
+
+
+def genomic_set_update(_pk):
+    """
+    Generate GenomicSetMember resource record.
```

Comment:
It seems like this should say "GenomicSet resource record" not "GenomicSetMember" since the code is using the `GenomicSetSchemaGenerator()`. It's not a big deal, but I just wanted to note it here.